packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -159,12 +170,24 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
 
     nextState.log.push({ cid: commitData.cid, type: CommitType.SIGNED })
 
-    const content = state.next?.content ?? state.content
-    const metadata = state.next?.metadata ?? state.metadata
+    const oldContent = state.next?.content ?? state.content
+    const oldMetadata = state.next?.metadata ?? state.metadata
+
+    const newContent = jsonpatch.applyPatch(oldContent, payload.data).newDocument
+    const newMetadata = { ...oldMetadata, ...payload.header }
+
+    if (newMetadata.schema) {
+      // TODO: SchemaValidation.validateSchema does i/o to load a Stream.  We should pre-load
```

Comment:
so, this is in many ways a step backward.  We've been working to get network i/o out of the commit application path by pre-loading the data required into `CommitData`, but here I'm pushing network i/o down into a function that's already been CommitData-ified.  I'm disappointed to do this, but I looked into trying to proactively load the schema contents into the CommitData at a higher level, but there are so many circular dependencies it quickly spiraled out of control.  In the interest of unblocking the Model and ModelInstanceDocument streamtype asap, I think doing this for now is worth it.

I do think, however, that the need for a larger refactor of stream loading and commit application is becoming more and more urgent by the day.